alpha/alpha-fsm/src/main/java/org/apache/servicecomb/pack/alpha/fsm/model/SagaData.java

```diff
@@ -34,6 +35,8 @@
   private Date endTime;
   private String globalTxId;
   private Date expirationTime;
+  private int timeout;
+  private SuspendedType suspendedType = SuspendedType.NULL;
```

Comment:
What's the meaning of NULL.

Reply:
> What's the meaning of NULL.

The value of suspendedType is SuspendedType.NULL when the transaction is not in the SUSPENDED state, Or SuspendedType.NONE is clearer?

Reply:
Yeah， NONE is much better.